
<meta charset="utf-8"/>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    // Don't process any delimiters, only <script type="math/tex">
    // tags generated by kramdown from $$...$$ in source.
    // (Could also avoid loading tex preprocessor - only need
    // tex input jax - but not worth the trouble.)
    tex2jax: {
      inlineMath: [],
      displayMath: [],
    }
  });
</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_HTML"></script>



## Class 1: Basic principles

### Jacob and Monod, J. Mol. Biol. 3: 318-356 (1961) [\[pdf\]](pdfs/jacob_monod.pdf)

_A seminal work of innovative synthesis, providing the intellectual foundation for the study of gene regulation._  

This work has played such a large role in the conceptualization of gene regulatory phenomena extending over many decades that we will be stepping through the key points of the paper sequentially.  The high-level themes/questions to consider are:

1. What is the relative importance assigned to positive (activator) vs. negative (repressor) regulation?
2. What logic supports Jacob and Monod’s conception of ‘messenger RNA’?
3. How do kinetic (vs. qualitative yes/no) experiments influence how Jacob and Monod conceptualize and development key problems of gene regulation?

### Rob Phillips, Annu. Rev. Condens. Matter Phys.  2015. 6:85–111 (2015) [\[pdf\]](pdfs/phillips.pdf)

_A modern physicists' view of transcriptional regulation_

This review describes a mathematical framework for building quantitative models of transcription.

1. How are the values of $$\Delta\epsilon_{rd}$$ shown in Figure 7 determined?
2. How many parameters (e.g. constants) does equation 16 depend on? How would adding or removing an additional repressor change this number?


------

## Class 2: Chromatin accessibility

### Ptashne, Nature 214:232-5 (1967) [\[pdf\]](pdfs/ptashne_nature_1967.pdf)

_This paper established that regulatory protein entities such as those discussed by Jacob and Monod were in fact DNA binding proteins._

1. Diagram the experiment
2. What are the experimental conditions and how could alterations in those conditions have affected the results?

------

### Weintraub and Groudine, Science 193:848-56 (1976) [\[pdf\]](pdfs/weintraub_groudine.pdf)

_This seminal paper provided the first evidence for the role of chromatin structure in gene expression._

1. How do the authors introduce DNaseI into the nucleus?
2. Describe the DNase I sensitivity assay as employed by the authors.  Could this be applied to any gene?
3. What do the authors. results suggest about the general organization of mammalian chromatin?
4. What prediction do the authors make concerning the spacial properties of gene expression in the nucleus?

------

### Galas and Schmitz Nucl. Acids Res. 5:3157-3170 (1978) [\[pdf\]](pdfs/galas_schmitz.pdf)
_This paper described a technique of fundamental importance for the characterization of promoters and other regulatory elements -- essentially allowing the combination of the type of information described by Jacob/Monod and Ptashe._ 

1. Describe the experiments and results illustrated in Figs. 2 and 5
2. Is the technique dependent on the location of a protein binding site within the test sequence?
NOTE:  We will be coming back to the kinetic analyses in Section C at a later class.

<!-- 
------

## Class 3: Distal regulatory elements, Part I

### Stalder et al.  Cell  20:451-460 (1980) [\[pdf\]](pdfs/stalder.pdf)

_This paper was the first to describe the chromatin architecture of mammalian genes in relation to their cis-regulatory elements.  The paper was also the first to show that regulatory sequences undergo chromatin structural changes in different cell types._

1. Diagram the experiment
2. Contrast the DNaseI sensitivity assays employed in this paper vs. those of Weintraub and Groudine (Science, 1976).  Describe the different levels of DNaseI sensitivity the authors report and their interpretations for the structure of gene domains.
3. How do the reported assays depend on the selection of restriction fragments and probes?
4. Describe the experiment and results of Figure 5
5. What significance to the authors assign to their results, and what predictions do they make?

------

### Banerji et al. Cell 27:299−308 (1981) [\[pdf\]](pdfs/banerji_1981.pdf)
_Discovery of transcriptional enhancers encoded in viral genomes._

1. What is the experimental schema and why did they devise it?
2. How are the 5’ transcript ends determined in Figure 2? How did they estimate the number of transcripts per cell?
3. How do they show that the enhancement phenomenon is not an artifact of the experimental system?
4. How well did they localize the sequences responsible for enhancer activity?

------

## Class 4: Distal regulatory elements, Part II

### Banerji et al.  Cell  33:729-40 (1983) [\[pdf\]](pdfs/banerji_1983.pdf)
_Discovery of the first enhancer of a tissue-specific eukaryotic cellular gene._

1. What is the definition of an enhancer element (i.e., specific criteria)? Outline the experiments that Banerji et al. used to fulfill these criteria.
2. Describe the experiments summarized in Figure two and Table two.
3. How does the enhancer element they identified differ from viral enhancers such as that of SV40?
4. What happens to this enhancer during B-cell maturation?

------

### Hanahan Nature 315:115-22 (1985) [\[pdf\]](pdfs/hanahan.pdf)
_First demonstration of the activity of a cellular enhancer in vivo._

1. What is an oncogene? What is hyperplasia?
2. Describe the experimental schema, including constructs and the breeding strategy
3. What can we conclude about cell and tissue specificity of transgene expression?
4. What phenotypes arise from the hybrid oncogenes?

------

## Class 3: Regulatory signals in DNA elements

### Pelham Cell 30:517-528 (1982) [\[pdf\]](pdfs/pelham.pdf)
_This paper provided the first evidence of distinct, environmentally-responsive regulatory sequences in the non-coding DNA of higher eukaryotes._

1. What does the experiment illustrated in Figure 2 establish?
2. Describe the experiment illustrated in Figure 4 and its main conclusion
3. Does HSE act as a thermostat? 

------

### Myers et al.   Science 232:613-619 (1986) [\[pdf\]](pdfs/myers.pdf)
_This paper describes the first large-scale empirical characterization of human promoter elements with a functional assay._ 

1. What are the CAAT, CACCC, and TATA boxes?  What proportion of human genes have these so-called "canonical" features?
2. Describe the enhancer-dependent transcription assay used by Myers et al to ascertain the effect of promoter point mutants.
3. Describe the assay used by Myers et al in Figure 2


------


## Class 4: Splicing

### Burnet et al. PNAS (1977) [\[pdf\]](pdfs/burnet.pdf)
_Discovery of RNA splicing_

1. What is an R-loop? How does R-loop mapping work?
2. Describe the results of the experiments shown in Figure 4. What can the authors conclude about the structure of the hexon gene?

------

### Tsai et al. Cell 22 (1 Pt 1):219–30 (1980) [\[pdf\]](pdfs/tsai.pdf)
_Demonstration that pre-mRNAs (so called hnRNAs) are the precursors for eukaryotic mRNA_

1. Describe the pulse-chase experimental design.
2. Explain the loop counting assay. How do the authors reach their conclusions about processing order from the data shown in Table 1?
3. What is the significance of the experiment discussed in Figure 10?

------ 



## Class 7: Distal regulatory elements, Part III

### Thanos and Maniatis  Cell  83:1091-100 (1995) [\[pdf\]](pdfs/thanos_maniatis.pdf)
_One of the first systematic descriptions of the fine structure of enhancer._

1. Describe the experiments in Figsure 3 and 4 and how these led to the proposed “enhanceosome” structure.
2. What evidence do they provide for cooperativity between the constituent regulatory elements within the enhanceosome?
3. What experiments suggest the existence of protein-protein interactions between NFkB and other components? -->


<!-- 
### Treisman and Maniatis Nature (1985) [\[pdf\]](pdfs/treisman_maniatis.pdf)
_First experiments aimed at discovering the mechanism of action of enhancers._

1. Describe the experimental strategy used by Treisman and Maniatis. Compare results from Figure 2 and 3. What does this comparison say about the magnitude of the enhancer effect in nuclear run-on vs. steady state RNA assays?
2. Describe the results of Figure 4. 
3. How do the authors rule-out non-specific initiation? 
-->

<!-- 
------

### Weintraub Proc. Natl. Acad. Sci. USA (1988) [\[pdf\]](pdfs/weintraub_1988.pdf)
_A seminal paper providing an elegant unifying model for the phenomenology of cis-acting transcriptional enhancers._

1. Diagram the experimental strategy of Weintraub. Summarize the qualitative results from Figure 2 and the related quantitative data reported in the text.
2. How did Weintraub show that only a limited number of active templates are expressed after transfection? Why is this important?
3. Describe the ‘dual marker’ experiment. What is the significance of this experiment and how is it exploited?

------



## Class 8: Distal regulatory elements, Part IV

### Grosveld et al. Cell 51:975-85 (1987) [\[pdf\]](pdfs/grosveld.pdf)
_Discovery of what we now call Locus Control Regions._

1. Describe the experimental schema. How does exogenous DNA integrate into the mouse genome?
2. What are position effects, and what is the significance of copy number-dependent expression?
3. What model is proposed by the authors to explain their observations?

### Blom van Assendelft et al Cell 56:969-977 (1989) [\[pdf\]](pdfs/blom_van_assendelft.pdf)
_First demonstration that the cell type-specific activity of a distal regulatory element is distinct from that of its promoter targets._

1. What is the structure of the experimental test constructs before and after transfection?
2. How do the authors test for copy number-dependent expression?
3. What are the general implications of the authors’ findings for the organization of genes within the genome?



------

## Class 9: Epigenetics and reprogramming

### Lee et al.  Cell  72:73-84 (1993) [\[pdf\]](pdfs/lee.pdf)
_First evidence suggesting a direct role for histone modification in transcriptional regulation._

1.	What property of the 5S RNA gene promoter makes it a model system for studying nucleosome-DNA interactions in vitro?
2.	Explain the results of Figure 3 and how they relate to the model shown in Figure 4
3.	Combining data from Figure 6 with those of Figure 3, what can we conclude about the nature of histone-DNA contacts from acetylated vs. non-acetylated histones?

------

### Davis et al. Cell 51:987-1000 (1987) [\[pdf\]](pdfs/davis.pdf)
_First demonstration that a transcription factor can reprogram cell fate._

1.	Describe subtractive cDNA hybridization / cloning and how it is applied in their experiments
2.	What can be concluded from the experiments shown in Figs. 2 and 3?
3.	Interpret the results of the experiment shown in Fig. 4 and the outcomes in Table 1 

-->

